account_financial_risk/models/res_partner.py

```diff
@@ -0,0 +1,540 @@
+        max_date = self._max_risk_date_due()
+        company_domain = self._get_risk_company_domain()
+        fields = [
+            "partner_id",
+            "account_id",
+            "amount_residual",
+            "amount_residual_currency",
+        ]
+        groupby = ["partner_id", "account_id", "currency_id"]
+        return {
+            "draft": {
+                "domain": company_domain
+                + [
+                    ("move_id.move_type", "in", ["out_invoice", "out_refund"]),
+                    ("account_type", "=", "asset_receivable"),
+                    ("parent_state", "in", ["draft", "proforma", "proforma2"]),
+                ],
+                "fields": fields,
+                "group_by": groupby,
+            },
+            "open": {
+                "domain": company_domain
+                + [
+                    ("reconciled", "=", False),
+                    ("account_type", "=", "asset_receivable"),
+                    "|",
+                    "&",
+                    ("date_maturity", "!=", False),
+                    ("date_maturity", ">=", max_date),
+                    "&",
+                    ("date_maturity", "=", False),
+                    ("date", ">=", max_date),
+                    ("parent_state", "=", "posted"),
+                ],
+                "fields": fields,
+                "group_by": groupby,
+            },
+            "unpaid": {
+                "domain": company_domain
+                + [
+                    ("reconciled", "=", False),
+                    ("account_type", "=", "asset_receivable"),
+                    "|",
+                    "&",
+                    ("date_maturity", "!=", False),
+                    ("date_maturity", "<", max_date),
+                    "&",
+                    ("date_maturity", "=", False),
+                    ("date", "<", max_date),
+                    ("parent_state", "=", "posted"),
+                ],
+                "fields": fields,
+                "group_by": groupby,
+            },
+        }
+
+    @api.depends(
+        "move_line_ids.amount_residual",
+        "move_line_ids.date_maturity",
+        "company_id.invoice_unpaid_margin",
+    )
+    def _compute_risk_account_amount(self):
+        self.update(
+            {
+                "risk_invoice_draft": 0.0,
+                "risk_invoice_open": 0.0,
+                "risk_invoice_unpaid": 0.0,
+                "risk_account_amount": 0.0,
+                "risk_account_amount_unpaid": 0.0,
+            }
+        )
+        customers = self.filtered(
+            lambda p: p == p.commercial_partner_id
+            or (p._origin and p._origin.id in p.commercial_partner_id.ids)
+        )
+        if not customers:
+            return  # pragma: no cover
+        groups = self._risk_account_groups()
+        for _key, group in groups.items():
+            group["read_group"] = self.env["account.move.line"].read_group(
```

Comment:
Use _read_group, same that:
https://github.com/OCA/credit-control/pull/377#discussion_r1587498744

Reply:
Yes, I had it in mind, but it passed me.
Changes done.